### PR TITLE
Featue/error

### PR DESCRIPTION
**Pull request by Shariefie**
check the error

### CODE REVIEW

**Review by Tjerk258 — APPROVED** [review]

File: Core/Src/API.c [hunk 1]
```diff
@@ -287,11 +287,14 @@ void drawFigure(uint8_t kleur, uint8_t nr_pointsgiven,...)
...
 {
-	if (nr > strlen((const char*)bitmaps))
+	if ((nr > strlen((const char*)bitmaps)) || ((0 < x_1up) && (x_1up > *bitmap)) || ((0 < y_1up) && (y_1up > *bitmap)))
 	{
 		softonErrorHandler(ERROR_BITMAP_NOT_FOUND);
 	}
-	//if (x_1up )
+	if (((x_1up+(uint16_t)(bitmap[BITMAP_WIDTH_LOW])) > VGA_DISPLAY_X) && ((y_1up + (uint16_t)(bitmap[BITMAP_WIDTH_HIGH] > VGA_DISPLAY_Y))))
+	{
+		softonErrorHandler(ERROR_BITMAP_OUT_OF_RANGE);
```
Comment (bestehustudentEU) [inline Core/Src/API.c:296]: comments?

File: Core/Src/front_layer.c [hunk 2]
```diff
@@ -75,6 +75,21 @@ void softonErrorHandler(uint8_t error)
 		puts("The number of bitmap you have chosen doesn't exist!");
 		break;
 	case ERROR_FIGURE_OUT_OFF_RANGE:
+		puts("Chosen fonts doesn't exist!");
```
Comment (bestehustudentEU) [inline Core/Src/front_layer.c:78]: comments?

File: Core/Src/API.c (hunk 1 above)
Comment (Shariefie) [reply Core/Src/API.c:296]: Solved.

File: Core/Src/front_layer.c (hunk 2 above)
Comment (Shariefie) [reply Core/Src/front_layer.c:78]: Solved.

**Pull request merged**